tests/mock_tests/portsorch_ut.cpp

```diff
@@ -675,6 +675,7 @@ namespace portsorch_test
         _unhook_sai_queue_api();
     }
 
+    /*
```

Comment:
This is temporary change until i fix this mock_test.

Reply:
will you fix in this PR?

Reply:
Yes. working on it.